float64[] features
---
float64 delta_h
